### PR TITLE
CB-115 : Fix Inconsistent result when random values passes for patient age.

### PR DESCRIPTION
**Pull request by paulupendo**
[Chore]Add logout link

# JIRA TICKET NAME: CB-115
[Click here to view ticket on JIRA](https://issues.openmrs.org/browse/CB-115?filter=17752)
## SUMMARY:
- The search returned inconsistent results due to a missing string to number conversion.
- Fixing the conversion returned consistent results.
- A logout link has also been added to CB OWA.

### CODE REVIEW

**Comment by wanjikum** [comment]
That's nice. Good work for activating the log out button

File: app/js/components/common/Header.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ export class Header extends Component {
 
                 <Link to="" activeClassName="active">
                     <li>
-                        <a href="#">Logout {' '}
+                        <a href="/openmrs/appui/header/logout.action?successUrl=openmrs">Logout {' '}
```
Comment (dkayiwa) [inline app/js/components/common/Header.js]: Is this change related to the ticket?

Comment (paulupendo) [reply app/js/components/common/Header.js]: The module lacked a logout link, hence the logout button was not working. To avoid raising a new ticket to add a single line link, we thought is best to include it in this PR and add it to the PR description. @dkayiwa

Comment (dkayiwa) [reply app/js/components/common/Header.js]: Why do you want to avoid creating a new ticket for something totally different from what this pull request was all about?

Comment (paulupendo) [reply app/js/components/common/Header.js]: Okay. Lets do the new ticket.
